sdk-base/src/main/java/com/mapbox/maps/plugin/ViewPlugin.kt

```diff
@@ -34,4 +34,14 @@ fun interface ViewPlugin : MapPlugin {
    * @param pixelRatio the device pixel ratio
    */
   data class Initializer(val context: WeakReference<Context>, val attributes: TypedArray, val pixelRatio: Float)
+
+  /**
+   * Static methods and variables.
+   */
+  companion object {
+    /**
+     * All view plugins are added as child views to the MapView using this constant as [View.setTranslationZ].
+     */
+    const val VIEW_PLUGIN_Z_TRANSLATION = 2.0f
```

Comment:
As discussed, this could break user's own views visibility. Negative translations should work fine and instead of [0;2] values we could use [-2;0] or similar. 
Maybe even API could be used to control this translation from the user side

Reply:
needs verification